src/main/java/com/rakovets/course/java/core/practice/io/file_analyzer/FileAnalyzeUtil.java

```diff
@@ -0,0 +1,154 @@
+        List<String> strList = getListOfFile(path);
+        String[] strings = strList.toString().split("\\W+");
+        for (String string : strings) {
+            if (string.matches("\\w+")) {
+                map.compute(string, (k, v) -> v == null ? 1 : v + 1);
+            }
+        }
+        Map<String, Integer> sortedMap = new LinkedHashMap<>();
+        List<Map.Entry<String, Integer>> mapList = new ArrayList<>(map.entrySet());
+        mapList.sort(Map.Entry.comparingByValue());
+        for (Map.Entry<String, Integer> entries : mapList) {
+            sortedMap.put(entries.getKey(), entries.getValue());
+        }
+        return sortedMap;
+    }
+
+    public static void getIntFile(Path path) {
+        List<String> intList = getListOfFile(path);
+        List<Integer> sortedIntList = new ArrayList<>();
+        String[] strings = intList.toString().split("\\D");
+        for (String str : strings) {
+            if (str.matches("\\d+"))
+                sortedIntList.add(Integer.parseInt(str));
+        }
+        sortedIntList.sort(Integer::compareTo);
+        try (FileOutputStream fos = new FileOutputStream(path + "v1")) {
+            byte[] buffer = sortedIntList.toString().getBytes(StandardCharsets.UTF_8);
+            fos.write(buffer, 0, buffer.length);
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    public static Map<String, String> getStudentPerfomance(Path path) {
+        Map<String, String> perfomanceMap = new LinkedHashMap<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(String.valueOf(path)))) {
+            String currentLine;
+            while ((currentLine = bufferedReader.readLine()) != null) {
+                String[] studentArray = currentLine.split("[,]");
+                double sum = 0;
+                for (int i = 1; i < studentArray.length; i++) {
+                    sum += Integer.parseInt(studentArray[i]);
+                }
+                double average = sum / (studentArray.length - 1);
+                BigDecimal averageRound = new BigDecimal(average);
+                averageRound = averageRound.setScale(2, RoundingMode.DOWN);
+
+                for (int i = 0; i < 1; i++) {
+                    perfomanceMap.put(studentArray[i], averageRound.toString());
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return perfomanceMap;
+    }
+
+    public static void changeJavaFileModifier(Path path, String fromModifier, String toModifier) {
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(String.valueOf(path)));
+            PrintWriter writer = new PrintWriter(
+                    new BufferedOutputStream(new FileOutputStream(path + "v1")));
+            String str;
+            while ((str = reader.readLine()) != null) {
+                str = str.replace(fromModifier, toModifier);
```

Comment:
This isn't correct because it will rename method `privateMyNameMethod` to `publicMyNameMethod`